src/applications/vaos/services/location/index.js

```diff
@@ -153,34 +174,54 @@ export async function getLocationSettings({ siteIds }) {
  * @async
  * @param {Object} params
  * @param {Array<string>} params.siteIds A list of 3 digit site ids to retrieve the settings for
+ * @param {boolean} params.useV2 Use the VAOS v2 endpoints to get location info
  * @returns {Array<Location>} An array of Locations with settings included
  */
-export async function getLocationsByTypeOfCareAndSiteIds({ siteIds }) {
+export async function getLocationsByTypeOfCareAndSiteIds({
+  siteIds,
+  useV2 = false,
+}) {
   try {
     let locations = [];
+    let settings = [];
 
-    const settings = await getLocationSettings({
-      siteIds,
-    });
-
-    const uniqueIds = settings.map(setting => setting.id);
-
-    // The above API calls only return the ids. Make an additional
-    // call to getLocations so we can get additional details such
-    // as name, address, coordinates, etc.
-    if (uniqueIds.length) {
+    if (useV2) {
```

Comment:
The logic for V2 is flipped around. We have to call the facilities endpoint to get all the child facilities for the registered sites, then we pass those ids to the scheduling configurations endpoint to get the VATS settings.